src/pretix/static/pretixcontrol/js/ui/dragndroplist.js

```diff
@@ -1,48 +1,57 @@
+        const container = $(this),
             url = container.data("dnd-url"),
             handle = $('<span class="btn btn-default btn-sm dnd-sort-handle"><i class="fa fa-arrows"></i></span>');
 
         container.find(".dnd-container").append(handle);
-        if (container.find("[data-dnd-id]").length < 2) {
+        if (!sessionStorage.dndShowMoveButtons) {
+            container.find(".sortable-up, .sortable-down").addClass("sr-only").on("click", function () {
+                sessionStorage.dndShowMoveButtons = 'true';
+            });
+        }
+        if (container.find("[data-dnd-id]").length < 2 && !container.data("dnd-group")) {
             handle.addClass("disabled");
             return;
         }
-
+        function maybeShowSortButtons() {
+            if (Date.now() - lastClick < 3000) {
+                $("[data-dnd-url] .sortable-up, [data-dnd-url] .sortable-down").removeClass("sr-only");
+            }
+            lastClick = Date.now();
+        }
+        container.find(".dnd-sort-handle").on("mouseup", maybeShowSortButtons);
+        const group = container.data("dnd-group");
+        const containers = group ? container.parent().find('[data-dnd-group="' + group + '"]') : container;
         Sortable.create(container.get(0), {
             filter: ".sortable-disabled",
             handle: ".dnd-sort-handle",
+            group: group,
             onMove: function (evt) {
                 return evt.related.className.indexOf('sortable-disabled') === -1;
             },
             onStart: function (evt) {
-                container.addClass("sortable-dragarea");
+                containers.addClass("sortable-dragarea");
                 container.parent().addClass("sortable-sorting");
+                didSort = false;
             },
             onEnd: function (evt) {
-                container.removeClass("sortable-dragarea");
+                containers.removeClass("sortable-dragarea");
                 container.parent().removeClass("sortable-sorting");
-
-                var disabledUp = container.find(".sortable-up:disabled"),
-                    firstUp = container.find(">tr[data-dnd-id] .sortable-up").first();
-                if (disabledUp.length && disabledUp.get(0) !== firstUp.get(0)) {
-                    disabledUp.prop("disabled", false);
-                    firstUp.prop("disabled", true);
-                }
-
-                var disabledDown = container.find(".sortable-down:disabled"),
-                    lastDown = container.find(">tr[data-dnd-id] .sortable-down").last();
-                if (disabledDown.length && disabledDown.get(0) !== lastDown.get(0)) {
-                    disabledDown.prop("disabled", false);
-                    lastDown.prop("disabled", true);
+                if (!didSort) {
+                    maybeShowSortButtons();
+                } else {
+                    $("[data-dnd-url] .sortable-up, [data-dnd-url] .sortable-down").addClass("sr-only");
```

Comment:
I am not sure, we want to hide the sort buttons again once one has sorted by drag-n-drop. On the other hand, there is no way to hide the sort-buttons again thanks to sessionStorage. So I guess we leave it like that?

Reply:
naja die idee wäre wenn du doch drag-und-drop bedienen kannst, brauchst du wahrscheinlich diese buttons doch nicht... und das mit dem sessionStorage finde ich schon sinnvoll weil du sie sonst nach jeder einzelnen sortieraktion wieder neu einblenden musst